src/Rings/mpoly.jl

```diff
@@ -13,9 +13,9 @@ import Hecke
 import Hecke: MapHeader, math_html
 
 export PolynomialRing, total_degree, degree, MPolyElem, ordering, ideal,
-       groebner_basis, eliminate, syzygy_generators, coordinates, 
-       jacobi_matrix, jacobi_ideal, radical, normalize, AlgebraHomomorphism,
-       divrem, primary_decomposition, isprimary, isprime
+       groebner_basis, eliminate, syzygy_generators, coordinates,
+       jacobi_matrix, jacobi_ideal, radical, divrem, primary_decomposition,
+       isprimary, isprime
```

Comment:
We need `normalize, AlgebraHomomorphism,` in this list.

Reply:
Git must have removed them after rebase was finished. I kept them when I was dealing with the rebase errors by hand...